anitya/templates/api.html

```diff
@@ -1,6 +1,6 @@
 {% extends "master.html" %}
 
-{% block title %} API | Anitya {% endblock %}
+{% block title %} API &mdash; Anitya {% endblock %}
```

Comment:
This is personal preference, but can we use a narrower separating character here? Tabs can get pretty narrow and space is a premium. I got 2 more characters on my Firefox tab using the middle dot character, ``·``.